Makefile

```
@@ -771,6 +767,145 @@ otel-forward: ## Port-forward Grafana (3000)
+	@kubectl wait --for=condition=Available deployment/perf-mock-server -n mcp-test --timeout=60s
+	@echo "Waiting for MCPServerRegistration..."
+	@kubectl wait --for=condition=Ready mcpsr/perf-mock-server -n mcp-test --timeout=120s
+	@echo "Perf test environment ready"
+
+.PHONY: perf-teardown
+perf-teardown: ## Remove perf test resources
+	kubectl delete -f tests/perf/manifests/registration.yaml --ignore-not-found
+	kubectl delete -f tests/perf/manifests/mock-server.yaml -n mcp-test --ignore-not-found
+
+PERF_OUT_DIR ?= out/perf/$(shell date +%Y%m%d-%H%M%S)
+PERF_PPROF_URL ?= http://localhost:6060
```

checkmake (0.3.2)

[warning] Variable "PERF_OUT_DIR" possibly contains a timestamp and should be simply expanded.

(timestampexpanded)